src/kinvey.ts

```diff
@@ -0,0 +1,45 @@
+import { Promise } from 'es6-promise';
+import url from 'url';
+import {
+  Kinvey as CoreKinvey,
+  isDefined,
+  KinveyError,
+  CacheRequest,
+  RequestMethod
+} from 'kinvey-js-sdk/dist/export';
+import { Client } from './client';
+
+const USERS_NAMESPACE = 'user';
+const ACTIVE_USER_COLLECTION_NAME = 'kinvey_active_user';
+
+interface ClientConfig {
```

Comment:
I was getting a typescript error complaining that the config object passed to `init` did not have some properties. I agree that I would rather not have it here. For the meantime, I can remove it and mark the config object as type `any`.